### PR TITLE
Startup screen

### PR DESCRIPTION
**Pull request by Markvanderuit**
Now shows a small startup/load screen initially.

If I forgot to plug a hole somewhere in the controller, feel free to say so.

### CODE REVIEW

File: src/main/java/control/FileMenuController.java [hunk 1]
```diff
@@ -90,13 +89,18 @@ public void save() {
...
+            temp = ScriptingProject.read(file);
+        } catch (Exception e) {
+            log.error("previously opened file could not be found.");
+        }
         if (temp == null) {
-            Alert alert = new Alert(AlertType.ERROR);
-            alert.setTitle("Load Failed");
-            alert.setContentText("The format in the selected file was not recognized");
-            alert.showAndWait();
+            System.out.println("dedede");
```
Comment (geenen124) [inline src/main/java/control/FileMenuController.java]: Please remove this print statement

File: src/main/java/gui/modal/StartupModalView.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import lombok.Getter;
+
+public class StartupModalView extends ModalView {
+
+    /*
+     * Tweakable styling variables.
```
Comment (mennoo1996) [inline src/main/java/gui/modal/StartupModalView.java:16]: Do we really need a multiline comment here?

File: src/main/java/gui/modal/StartupModalView.java [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+     */
+
+    // width and height of screen. 680 and 350 work very, very well.
+    private static final int width = 550;
```
Comment (mennoo1996) [inline src/main/java/gui/modal/StartupModalView.java:20]: Update comment to match actual values

File: src/main/java/gui/modal/StartupModalView.java (hunk 2 above)
Comment (Markvanderuit) [inline src/main/java/gui/modal/StartupModalView.java:16]: Big and noticeable. I prefer it this way for all touchable/non-touchable variables.

File: src/main/java/gui/modal/StartupModalView.java (hunk 3 above)
Comment (Markvanderuit) [inline src/main/java/gui/modal/StartupModalView.java:20]: Fixed/removed.

**Comment by mennoo1996** [comment]
Good work!

**Pull request merged**